test/core/util/test_config.cc

```diff
@@ -135,26 +97,42 @@ static void print_stack_from_context(CONTEXT c) {
 #endif
 
   HANDLE process = GetCurrentProcess();
-  HANDLE thread = GetCurrentThread();
 
   SYMBOL_INFOW* symbol =
       (SYMBOL_INFOW*)calloc(sizeof(SYMBOL_INFOW) + 256 * sizeof(wchar_t), 1);
   symbol->MaxNameLen = 255;
   symbol->SizeOfStruct = sizeof(SYMBOL_INFOW);
 
-  while (StackWalk(imageType, process, thread, &s, &c, 0,
-                   SymFunctionTableAccess, SymGetModuleBase, 0)) {
-    BOOL has_symbol =
-        SymFromAddrW(process, (DWORD64)(s.AddrPC.Offset), 0, symbol);
-    fwprintf(
-        stderr, L"*** %016I64X %ls - %016I64X\n", (DWORD64)(s.AddrPC.Offset),
-        has_symbol ? symbol->Name : L"<<no symbol>>", (DWORD64)symbol->Address);
+  const unsigned short MAX_CALLERS_SHOWN = 32;
+  for (int frame = 0; frame < MAX_CALLERS_SHOWN && StackWalk(imageType, process, thread, &s, &c, 0,
+                   SymFunctionTableAccess, SymGetModuleBase, 0); frame++) {
+    SymFromAddrW(process, (DWORD64)(s.AddrPC.Offset), 0, symbol);
```

Comment:
nit: check the return value from `SymFromAddrW` for failure, and if possible log the `GetLastError()` in that case if so?

Reply:
Turns out the method returns false if the symbol could not be found, so I updated the code accordingly
(previously there was a bug that would display the previous frame's info if a symbol wasn't found)